dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/UDPUtils.java

```diff
@@ -0,0 +1,158 @@
+                    try {
+                        Thread.sleep(retryDelay);
+                    } catch (InterruptedException x) {
+                        // do nothing but debug
+                        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                            Tr.debug(tc, "sleep caught InterruptedException.  will proceed.");
+                        }
+                    }
+                    bindHelper(framework, bootstrap, inetHost, inetPort, retryCount - 1, retryDelay, bindListener);
+                } else {
+                    Tr.error(tc, UDPMessageConstants.BIND_FAILURE, new Object[] { channelName, inetHost, String.valueOf(inetPort) });
+                }
+            }
+        });
+        return bindFuture;
+    }
+
+    /**
+     * UDP start helper implementation 
+     * 
+     * @param framework
+     * @param bootstrap
+     * @param inetHost
+     * @param inetPort
+     * @param bindListener
+     * @return
+     * @throws NettyException
+     */
+    public static ChannelFuture start(NettyFrameworkImpl framework, BootstrapExtended bootstrap, String inetHost, int inetPort, ChannelFutureListener bindListener) throws NettyException {
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Tr.debug(tc, "start (UDP): attempt to bind a channel at host " + inetHost + " port " + inetPort);
+        }
+        try {
+            framework.runWhenServerStarted(new Callable<ChannelFuture>() {
+                @Override
+                public ChannelFuture call() throws NettyException {
+                    UDPConfigurationImpl config = (UDPConfigurationImpl) bootstrap.getConfiguration();
+                    int bindRetryCount = config.getRetryCount();
+                    int bindRetryInterval = config.getRetryInterval();
+                    InetSocketAddress address = null;
+                    String newHost = inetHost;
+                    if (newHost.equals("*")) {
+                        newHost = "0.0.0.0";
+                    }
+                    address = new InetSocketAddress(newHost, inetPort);
+                    if (address.isUnresolved()) {
+                        final String channelName = ((UDPConfigurationImpl) bootstrap.getConfiguration()).getExternalName();
+                        Tr.error(tc, UDPMessageConstants.DNS_LOOKUP_FAILURE, 
+                                new Object[] { channelName, newHost, String.valueOf(inetPort) });
+                        throw new NettyException("local address unresolved");
+                    }
+
+                    return bindHelper(framework, bootstrap, newHost, inetPort, bindRetryCount, bindRetryInterval, bindListener);
+                }
+            });
+        } catch (Exception e) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "caught exception performing late cycle server startup task: " + e);
```

Comment:
@Error scheduling bind of UDP channels on server startup?"

Reply:
this is actually handling the exception from `Callable.call()`, I think it's fine to leave it as-is